### PR TITLE
19754 Update amalgamation resources for Certify component

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#19754

*Description of changes:*
- app version = 5.8.13
- removed redundant first certify bullet
- set certify entity display to "business"

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/resources/AmalgamationRegular/BC.ts [hunk 1]
```diff
@@ -33,11 +33,10 @@ export const AmalgamationRegResourceBc: AmalgamationResourceIF = {
   reviewAndConfirm: {
     completingPartyStatement: {
       certifyStatements: [
-        ResourcePhrases.RELEVANT_KNOWLEDGE_OF_BUSINESS,
         ResourcePhrases.AMALGAMATION_CANNOT_BE_REVERSED
       ],
       certifyClause: ResourcePhrases.OFFENCE_SECTION_427,
-      entityDisplay: null
```
Comment (severinbeauvais) [inline src/resources/AmalgamationRegular/BC.ts:40]: Old:

image

New:

image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-653-2qjaz79j.web.app

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

**Pull request merged**